src/preferencesControl.ts

```diff
@@ -299,6 +311,19 @@ export class PreferencesControl {
             });
         }
     }
+
+    /**
+     * Remove outline class in radio button which is not focused
+     */
+    private removeBlurRadioBtnOutline(target: HTMLElement): void {
+        if (target) {
+            let radioBtnOutline = target.parentElement!;
+            let radioBtnOutlineClass = radioBtnOutline.className.lastIndexOf(' ');
```

Comment:
If tomorrow we add other styles to the end this code will stop working and we will totally forget about this. Need to have a more reliable way to remove a class.
One possible way `.replace(" class-name", "")`